servers/quarkus-server/pom.xml

```diff
@@ -168,6 +168,7 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <argLine>@{argLine} -Dfile.encoding=UTF-8 --add-opens=java.base/java.util=ALL-UNNAMED</argLine>
```

Comment:
imo we should add a comment to explain why we added this (and to know when we can remove it again)